README.md

```diff
@@ -82,92 +76,16 @@ workbook.add_worksheet do |sheet|
   sheet.add_border 'B2:D5'
   sheet.add_border 'B3:D3', [:top]
 end
-workbook.apply_styles
 axlsx.serialize 'grocery.xlsx'
 ```
 
-### `axlsx` gem without `axlsx_styler`
```

Comment:
This section was moved to a different file, since it doesn't directly relate to this gem.